db/db_flush_test.cc

```diff
@@ -247,8 +247,10 @@ TEST_F(DBFlushTest, ManualFlushFailsInReadOnlyMode) {
   ASSERT_OK(db_->ContinueBackgroundWork());
   dbfull()->TEST_WaitForFlushMemTable();
   uint64_t num_bg_errors;
+#ifndef ROCKSDB_LITE
```

Comment:
Maybe initialize `num_bg_errors` and put the assertion inside `ifndef ROCKSDB_LITE`.